content/docs/addons-test-utils.md

```diff
@@ -112,6 +112,10 @@ mockComponent(
 
 Pass a mocked component module to this method to augment it with useful methods that allow it to be used as a dummy React component. Instead of rendering as usual, the component will become a simple `<div>` (or other tag if `mockTagName` is provided) containing any provided children.
 
+> Note:
+>
+> `mockComponent()` is a legacy method that is no longer needed. A similar pattern can be implemented directly.
```

Comment:
How do you feel about the following alternative wording?

> `mockComponent()` is a legacy API. We recommend using [shallow rendering](/docs/test-utils.html#shallow-rendering) instead.

Reply:
shallow rendering or [`jest.mock()`](https://facebook.github.io/jest/docs/en/tutorial-react-native.html#mock-native-modules-using-jestmock) if you use Jest.

Reply:
Nice! I like that there's recommendations in this alternate wording -- added a commit :)